packages/angular_devkit/build_angular/package.json

```diff
@@ -75,6 +75,7 @@
   },
   "peerDependencies": {
     "@angular/compiler-cli": ">=10.1.0-next.0 < 11",
+    "@angular/localize": "^10.0.0-next.0",
```

Comment:
`10.0.x` is still supported if not using ivy extraction.
I'm also not sure why the  compiler-cli range is setup like that.  `^10.1.0-next.0` and `>=10.1.0-next.0 < 11` are equivalent.

Reply:
Yeah they are.

Maybe we can align both compiler-cli and localize to be in the same range?

Reply:
Same thing here: https://github.com/angular/angular-cli/blob/aacb98d85bac837b898dc6158459aef53f5569a3/packages/ngtools/webpack/package.json#L30

Reply:
I think for stable releases it should be just `^10.0.0` since tooling is only linked to the framework by major version.   But during prerelease periods it would need to be `^10.0.0 || ^10.1.0-next.0` (or whatever the current prerelease minor is) since the prerelease identifier only applies to the referenced major/minor/patch combination.

I'll use the same range as the compiler-cli here and then they all can be updated together, if decided, separately since that is really out of scope for this PR.

Reply:
I do agree that it should be `^10.0.0 || ^10.1.0-next.0`, but indeed it’s not in scope of this PR.